cli/command/manifest/fetch.go

```diff
@@ -0,0 +1,326 @@
+package manifest
```

Comment:
> This sounds like an argument for exporting things to make them easier to discover.

Not exactly: most people don't do this. In my experience, when I have found something I've needed, it would require a change that would break another use case. Making good stable interfaces is hard.

Analyze the packages in Docker. Many only have a single consumer and have no documented behavior. Whether or not these packages are exported has no bearing on reuse. The only thing that will help with reuse is effective pattern identification and ruthless refactoring. :hamster: